core/src/test/scala/examples/DummyConsumer.scala

```diff
@@ -1,4 +1,4 @@
-package com.softwaremill.react.kafka2
+package examples
```

Comment:
Is it okay to simplify the package name?

I don't mind it being long, but I think example (and test) package names are best being something else than the package itself.